inst/rmd/h/default.html

```diff
@@ -469,15 +529,20 @@
 $endfor$
 
 $if(theme)$
-<div class="fluid-row" id="$idprefix$header">
+$if(bs3)$
+<div id="$idprefix$header">
+$else$
+<!-- bs4 removed margin-top on h1, h2, ..., h6 -->
+<div id="$idprefix$header" style="margin-top: 20px">
```

Comment:
Should we add `margin-top` for all headings?? Or for `<section>`??

Reply:
Done in https://github.com/rstudio/rmarkdown/pull/1706/commits/69161c178ca7a3f59870e4cbd68ef428a2985239